pages/common/clj.md

```diff
@@ -0,0 +1,29 @@
+# clj
+
+> Clojure tool to start a REPL or invoke a specific function with data.
+> All options can be defined in a `deps.edn` file.
+> More information: <https://clojure.org/guides/deps_and_cli>.
+
+- Start a REPL:
+
+`clj`
+
+- Exec function:
+
+`clj -X {{namespace/fn_name}}`
+
+- Run -main function of a namespace:
+
+`clj -M -m {{namespace}} {{args}}`
+
+- Prepare (resolve deps, download libraries, make and cache classpaths):
```

Comment:
Perhaps this will read better?
```suggestion
- Prepare a project by resolving dependencies, downloading libraries, and making / caching classpaths:
```